test/500_weave_multi_cidr_test.sh

```diff
@@ -26,7 +26,15 @@ assert_zone_records() {
     FQDN=$3
     shift 3
 
-    records=$(weave_on $HOST status | grep "^$CID") || true
+    FORMAT="{{range .DNS.Entries}}\
+{{if eq \"$CID\" .ContainerID}}\
+{{if eq .Tombstone 0}}\
+{{.Hostname}} {{.Address}}
+{{end}}\
+{{end}}\
+{{end}}"
+
+    records=$(weave_on $HOST report -f "$FORMAT") || true
```

Comment:
This is all a bit unwieldy. Perhaps we should just use `weave dns-lookup`? That won't allow us to match on the container id, but I don't think that is so important.